pkg/errors/release.go

```diff
@@ -194,3 +194,24 @@ func NewNotWorkingOnStrategyError(contenderReleaseKey string) NotWorkingOnStrate
 		contenderReleaseKey: contenderReleaseKey,
 	}
 }
+
+type ContenderStrategyIncompleteError struct {
+	relKey string
+}
+
+func (e ContenderStrategyIncompleteError) Error() string {
+	return fmt.Sprintf(
+		"release contender %q strategy is incomplete yet",
```

Comment:
This reads weird. I'd suggest either "[...] is still incomplete" or "[...] is not complete yet".